www_src/main.less

```diff
@@ -42,7 +42,7 @@ body {
   height: 100%;
   margin: 0;
   padding: 0;
-  font-family: 'Roboto', sans-serif;
+  font-family: "FiraSans", "Fira Sans", "Fira Sans OT", sans-serif;
```

Comment:
Those are the three names used depending on which version of Fira is installed on the local system. This is a direct port from the `master` branch after we had various different problems with contributors and cross platform / browser issues with fonts.